docker-compose.yml

```
@@ -1,18 +1,9 @@
-      - '3000:3000'
-
-  playwright:
-    build:
-      context: .
-      dockerfile: Playwright.Dockerfile
-    restart: unless-stopped
-    volumes:
-      - .:/app
-    ports:
-      - '9323:9323'
+      - '3000:3000'
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)